.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
1. new line should be at the end of any file
2. what is this for?

Reply:
I have problems with a dependency tree and nothing else helps expect this line in .env file

Reply:
new line